### DISCUSSION TITLE
Temporary LOCK the state of a relay?

### DISCUSSION THREAD

**Discussion by stefanbode** [arendst/Tasmota#21050] [General]
For shutters there is a lock function to prevent children or Alexa to make fun with your shutters. Is there a general option like "LockRelay" to temporary freeze the current relay state so that buttons and switches do not work anymore?

PowerOnState 4 is only half of the equation. This keeps it on. But how to keep it OFF?

**Comment by sfromis** [comment]
The general approach would be to "take over" Button/Switch input states using rules, which can then manage what commands to issue, including to mimic the default actions without rules, or maybe do nothing. IDK if this would work well with shutters.

**Comment by stefanbode** [comment]
My main topic is to keep Alexa away from control if I am not at home. The commands coming from Alexa are MQTT. HA is the connection to Alexa. Adding a PowerOnState 6 seems to be the easiest but this is neither self explaining nor can you restrict it on dedicated relays.
